README.md

```
@@ -1 +1,138 @@
+## Usage
+
+1. Create a test file called test.yaml. See [csv_2_csv_example.yaml](data_validation_tests/csv_2_csv_example.yaml) for format.
+2. Execute the test, ```python main.py --config test.yaml```
+3. Review the results (Coming Soon)
+
+## Comparison Rules
+
+Cell Checker uses a rules engine, durable_rules, to perform comparisons between the two dataframes.
+Out of the box, the framework uses the following rules:
+
+- equal_row_count: Determines if the source and target data has the same number of rows
```

Markdownlint (0.37.0)

Expected: asterisk; Actual: dash
Unordered list style

(MD004, ul-style)